src/metabase/api/collection.clj

```diff
@@ -322,25 +322,28 @@
             :dataset_query)))
 
 (defn- card-query [dataset? collection {:keys [archived? pinned-state]}]
```

Comment:
The FE needs `database_id` for models in metabot, so it is being conditionally added. Question: Should we just add it in globally to the select (no cond->) or is there any reason to limit it to models only?

Reply:
do whatever you need. There is `post-process-collection-children` which can let you dissoc database_id from other collection items that don't need it, eg, for pulse:

```clojure
(defmethod post-process-collection-children :pulse
  [_ rows]
  (for [row rows]
    (dissoc row
            :description :display :authority_level :moderated_status :icon :personal_owner_id
            :collection_preview :dataset_query)))
```
Probably best to stay tidy and not put `database_id nil` on all items.